edx/edx-app-ios
# Whats New Feature

### Description

[LEARNER-185](https://openedx.atlassian.net/browse/LEARNER-185)

This PR implements new feature named 'Whats New'. This feature is was informing user to whats being delivered in latest version. 

### How to test this PR

This feature will appear on first launch after login or if user already then on opening app.
